cmd/broker/suite_test.go

```diff
@@ -512,391 +227,6 @@ kyma-template: |-
 	return resources
 }
 
-type ProvisioningSuite struct {
```

Comment:
this was the initial way of testing, but deprecated. all tests which were using this helper were migrated to BrokerSuiteTest